### ISSUE TITLE
Save icon shouldn't flash to offline on every page load

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-connectors#72]
This is a regression from the current version, which just changes to the next applicable icon. While that means that, for a slow page load, the icon can be briefly incorrect (and ideally we would [wait to start the save](https://github.com/zotero/zotero/issues/868) until it was done loading and the right translator was ready), going to offline on every load is very distracting.

**Comment by adomasven** [comment]
The intended behaviour here is to show Zotero status on the button while a page is loading.  What's going on here is the button is disabled until page load. Chrome makes a disabled button greyscale while Firefox keeps the colouration as is.

We could not use chrome provided button disabling and handle that ourselves, which would keep the button colouration and keep showing the client status on page load (which I think is nice). Or we can revert back to the previous behaviour.

Technically we are flashing/changing the icon for most page loads anyway, but perhaps this _is_ more distracting.

**Comment by dstillman** [comment]
I understand the reason — I'm just saying it's very distracting, and we should go back to the previous behavior.

**Comment by dstillman** [comment]
I experimented with this a bit when I took over the connector and decided not to disable the icon each time. That way the icon doesn't change at all between any two non-translator webpages, and it only changes once otherwise. This way it always changes twice (to disabled and back), which is much more annoying. (As I say, I think the icon should be mostly advisory, with the actual save delaying until a proper translator is decided, but that's a separate issue.)
